src/audio/mixer/mixer.c

```diff
@@ -100,8 +100,8 @@ static int mixer_process(struct processing_module *mod,
 	for (i = 0; i < num_input_buffers; i++) {
```

Comment:
What's with the "modAPI: " prefix ? Probably not worth changing now as it's in the original comment, but this is fails the "follow existing codebase style" sanity check quite badly.